tests/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "cordova-plugin-tests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "coordinateTransformTests.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha coordinateTransformTests.js"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Change to Apache v2 (see https://github.com/IndoorAtlas/cordova-plugin/blob/master/LICENSE)